scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/vfl/example/psi/PSIExample.scala

```diff
@@ -19,8 +19,8 @@ package com.intel.analytics.bigdl.ppml.vfl.example.psi
 import org.slf4j.LoggerFactory
 import java.util
 
-import com.intel.analytics.bigdl.ppml.psi.test.TestUtils
-import com.intel.analytics.bigdl.ppml.vfl.PSI
+import com.intel.analytics.bigdl.ppml.psi.HashingUtils
+import com.intel.analytics.bigdl.ppml.vfl.{PSI, VflContext}
```

Comment:
Is PSI part of VFL algorithms? I think we should make it `com.intel.analytics.bigdl.ppml.PSI` or `com.intel.analytics.bigdl.ppml.algorithms.PSI` instead.